CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## [Unreleased]
 
+## [1.2.2] - 2019-05-17
+
+ - Updated Facebook API version to 3.2
+ - Updated required googleads version to == 19.0.0
```

Comment:
```suggestion
 - Updated required googleads version to 19.0.0
```

Reply:
I'm keeping the same format we were using in the changelog, which makes sense when it's a `>= version`, but i don't mind only using it in that case